spec/models/notification_spec.rb

```diff
@@ -0,0 +1,49 @@
+# == Schema Information
+#
+# Table name: notifications
+#
+#  id         :bigint           not null, primary key
+#  details    :text
+#  viewed     :boolean          default(FALSE), not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  user_id    :bigint           not null
+#
+# Indexes
+#
+#  index_notifications_on_user_id  (user_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (user_id => users.id)
+#
+require 'rails_helper'
+
+RSpec.describe Notification, type: :model do
+  let!(:user) { create :user }
+  let!(:notification) { create :notification, user: }
```

Comment:
Since the `notification` factory already generates a user you should not need to define it, e.g. just have 
```ruby
let!(:notification) { create :notification }
```
If you need to refer to the user object in tests, then:
```ruby
let!(:notification) { create :notification }
let(:user) { notification.user }
```
however it doesn't look like need it.